api/src/pcapi/models/feature.py

```diff
@@ -130,6 +130,7 @@ class FeatureToggle(enum.Enum):
     ENABLE_PRO_NEW_NAV_MODIFICATION = "Activer la modification du statut de la navigation du portail pro"
     WIP_ENABLE_NEW_HASHING_ALGORITHM = "Activer le nouveau système de hachage des clés publiques d'API"
     WIP_ENABLE_BOOST_TWO_STAGES_BOOKING = "Activer la réservation Boost en 2 étapes."
+    WIP_BENEFICIARY_EXTRACT_TOOL = "Active la liste d'utilisateur ayant demandé leur données après anonymisation"
```

Comment:
Non, ce n’est pas après anonymisation ! Dans ce cas il n’y aurait plus de données !
```suggestion
    WIP_BENEFICIARY_EXTRACT_TOOL = "Activer l'extraction de données personnelles (RGPD)"
```